wordle/src/main/kotlin/domain/WordleGame.kt

```diff
@@ -0,0 +1,36 @@
+package domain
+
+class WordleGame(var round: Int = 0,
+                 var isEnd: Boolean = false,
+                 val gameResult: GameResult = GameResult()) {
+
+    val answer = Answer.todayAnswer()
+    private val answerChecker = AnswerChecker(answer)
+
+    fun proceed(word: Word) {
+        val roundResult = answerChecker.check(word)
+        round++
+        gameResult.appendResult(roundResult)
+
+        if (roundResult.isAnswer || round.isFinish()) {
+            isEnd = true
+        }
+    }
+
+    val isInProgress: Boolean
+        get() {
+            return !isEnd
+        }
```

Comment:
```suggestion
    val isInProgress: Boolean
        get() = !isEnd
```
더 간단하게 쓸 수 있을 것 같아요. 그리고 이것도 프로퍼티인데 함수보다 위에 선언하는게 좋지 않을까요?

Reply:
오 대박 👍